src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -120,12 +120,25 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
 
         // AFTER DELETE
         if (triggerAction == TDTM_Runnable.Action.AfterDelete && !TDTM_ProcessControl.getRecursionFlag(TDTM_ProcessControl.registeredTrigger.ACCT_IndividualAccounts_TDTM_After_Delete)) {
-            for (Contact c : oldContacts){
+            Set<Id> relatedAccountIds = new Set<Id>();
+            List<Account> relatedAccounts = new List<Account>();
+            for (Contact c : oldContacts) {
                 listContactAccountDelete.add(c);
+                //Collect related account ids
+                relatedAccountIds.add(c.AccountId);
+            }
+
+            //Re-query to get correct Account values
+            relatedAccounts = [SELECT Id,
+                                    RecordType.Id
```

Comment:
I think we can do SELEC Id, RecordTypeId FROM Account... here rather than RecordType.Id -- I believe the standard objects get a "RecordTypeId" field once we introduce recordtype on the object. Its one less table to traverse on the backend, probably not a big deal though. It would then also change "account.RecordType.Id to account.RecordTypeId in the logic. Just a thought.

That said perhaps you're aware of issues that I am unaware of. Both ways work so if you want to leave it, that is fine. It looks like line 165 uses that field even though it's coming from the contact "contact.Account.RecordTypeId"

Reply:
@lifewithryan those two ways should return the same value but I like the idea of using RecordTypeId!